docker/docker_image_src.go

```diff
@@ -384,7 +385,7 @@ func deleteImage(ctx context.Context, sys *types.SystemContext, ref dockerRefere
 	// When retrieving the digest from a registry >= 2.3 use the following header:
 	//   "Accept": "application/vnd.docker.distribution.manifest.v2+json"
 	headers := make(map[string][]string)
-	headers["Accept"] = []string{manifest.DockerV2Schema2MediaType}
+	headers["Accept"] = []string{manifest.DockerV2Schema2MediaType, imgspecv1.MediaTypeImageManifest}
```

Comment:
This works for that specific bug, but shouldn’t this use all of `manifest.DefaultRequestedManifestMIMETypes`?  Or maybe just call `fetchManifest` explicitly.

Reply:
Notably this will get out of sync the next time a MIME type is added again.

Reply:
Is there a function call to get the list of all mimetypes?

Reply:
This needs to be ~hard-coded in the client; what the server supports does not quite matter if the client can’t deal with it.

See above for the variable hard-coding it in c/image.

Reply:
Very good point, @mtrmac. I'll spin a new PR here and update https://github.com/containers/skopeo/pull/680.

Reply:
Eugh ... no idea what's going on. The last commit vndr is 11 days ago, so I can't be caused by some outdated caches in Travis. I think it's time for go modules :^)

Reply:
^ the comment was meant for another thread. Mixed up tabs...